presto-docs/src/main/sphinx/release/release-0.273.rst

```diff
@@ -0,0 +1,84 @@
+=============
+Release 0.273
+=============
+
+**Highlights**
+==============
+* Add new Presto connector for Clickhouse
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix disaggregated coordinator resource group aggregation issue related to ``WAITING_FOR_PREREQUISITE`` queries. When a query is moved to WAITING_FOR_PREREQUISITE, resource manager would end up counting them towards running queries. This gives a wrong impression to coordinator about running queries and it would end up running less queries than configured.
+* Fix a bug where ``EXPLAIN (TYPE IO)`` did not include the index source from an index join.
+* Fix a memory over-counting issue for using varchar types that could cause queries to fail with exceeded memory limit errors.
+* Fix an off-by-one error during buffer size calculation in :func:`tdigest_agg`.
+* Fix memory tagging for UnnestOperator and TableWriterMergeOperator.
+* Improve performance of :func: `zip_with` when used inside a :func:`reduce_agg`.
+* Add :func:`to_base32` and :func:`from_base32` varbinary functions.
+* Add support for ``TRUNCATE TABLE`` statement.
```

Comment:
`Add support for ``TRUNCATE TABLE`` statement and associated :doc:`/sql/truncate`.`